episodes/web-location-identifiers.Rmd

```diff
@@ -0,0 +1,257 @@
+
+The **Uniform Resource Identifier (URI)** is a string of characters formulated to uniquely identify a resource, (most commonly on the Web) and enable interaction with it via common protocols such as HTTP.
+
+A **Uniform Resource Name (URN)** is a type of URI. It is a standard, persistent and unique identifier for digital resources on the Internet. To link to the resource from the URN, a **resolver service is required.**
+
+The **Uniform Resource Locator (URL)** is a string of characters used to access the information or resource by **using the address of the resource location** via communication protocols such as HTTP. </span>
+<br><br>
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: callout 
+## How links rot
+
+![URL Example](fig/urlSyntax.png){alt="Examplary representation of the URL syntax: protocol specification, host adress, file path and query parameters."}
+
+The **URL** specifies the exact location of a retrievable resource on the web (host & path) as well as the communication protocol (e.g. HTTP) necessary for the local browser to address the host server.
+If you open a link on the web operated under the HTTP protocol, the browser sends a **GET request** to the host server, specifying the path to the requested resource. The host server then follows this path and responds with the resource information, if retrievable.
+
+A resource might **not be retrievable** by the host server, if:
+<ul>
+  <li> the location was misspelled.</li>
+  <li> the file path on the host server changed.</li>
+  <li> the resource was deleted.</li>
+  <li> the resource was migrated to a different server.</li>
+</ul>
+
+Each of these events will result in the **404 error** response. To avoid these broken or rotten links, assigning a **Persistent Identifier (PID)** to the resource resolves the issue.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+A **Persistent Identifier (PID)** is a long-lasting reference to digital objects such as websites, articles, datasets, persons or organizations. PIDs are **globally unique** and **persistent over time**. Assigning a PID to a digital resource ensures its permanent indentifiability, accessibility, and referenciability.[^16] <br><br>
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: callout
+## PID services
+
+A **PID service** operate as a persistent look-up service that redirects GET requests for digital resources to the latest URL of this resource.<br>
+**The host of the digital resource is responsible to update the resource's location with the PID service after the location changed.** <br><br>
+
+**Frequently used PIDs in the scientific context are:**
+<ul>
+  <li>Digital Object Identifier (<a href="https://www.doi.org/index.html" target="_blank">DOI</a>)</li>
+  <li>Persistent Uniform Resource Locator (PURL, e.g. <a href="https://purls.helmholtz-metadaten.de/" target="_blank">PIDA</a>)</li>
```

Comment:
> PURL, e.g. PIDA
This seems to me like PIDA is an example of a PURL. 
Also, PIDA is a service not a PID.